client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -137,11 +137,22 @@ const isRedirecting: Reducer< boolean, OnboardAction > = ( state = false, action
 	return state;
 };
 
+const hasUsedPlansStep: Reducer< boolean, OnboardAction > = ( state = false, action ) => {
```

Comment:
this new reducer has to be added to the persistence list in order to persist between 2 user visits: https://github.com/Automattic/wp-calypso/blob/b524c70eac79dfcbef7f5bbff1423fe48a232422/client/landing/gutenboarding/stores/onboard/index.ts#L27-L35

Reply:
excellent, thank you!